lib/game_completion.rb

```diff
@@ -0,0 +1,19 @@
+module TicTacToe
+  class GameCompletion 
+
+    def initialize(console_ui) 
+      @console_ui = console_ui
+    end
+
+    def game_over_messages(game_result, board_array)
```

Comment:
What do you think about making use of your GameEval object inside of this method? You've already implemented #won? and #tie? and you could also add logic to return the winning player's mark. This should give you the benefit of not needing to use string values to indicate the game result, as well as eliminating some duplication of logic